doc/languages-frameworks/rust.section.md

```diff
@@ -464,6 +464,8 @@ you of the correct hash.
   be disabled by setting `dontUseCargoParallelTests`.
 * `cargoInstallHook`: install binaries and static/shared libraries
   that were built using `cargoBuildHook`.
+* `rustBindgenHook`: for crates which use `bindgen` as a build dependency, lets
```

Comment:
```suggestion
* `bindgenHook`: for crates which use `bindgen` as a build dependency, lets
```
Repeating `rust` seems unnecessary. I don't feel strongly about it though.

Reply:
renamed to bindgenHook